fiscal_epos_print/static/src/js/chrome.js

```diff
@@ -94,6 +94,28 @@ odoo.define("fiscal_epos_print.chrome", function (require) {
         },
     });
 
+    var PrinterFiscalOpenCashDrawer = PosBaseWidget.extend({
+        template: 'PrinterFiscalOpenCashDrawer',
+
+        button_click: function () {
+            this.chrome.loading_show();
+            this.chrome.loading_message(_t('Connecting to the fiscal printer'));
+            var protocol = ((this.pos.config.use_https) ? 'https://' : 'http://');
+            var printer_url = protocol + this.pos.config.printer_ip + '/cgi-bin/fpmate.cgi';
+            var printer_options = {url: printer_url, requested_z_report: true};
```

Comment:
`requested_z_report` forse è un refuso

Reply:
si, lo rimuovo

Reply:
anzi! mi sà che dobbiamo rivedere un pò stà ``request_z_report`` perchè abbiamo anche questo blocco di codice che sicuramente andrà rivisto https://github.com/OCA/l10n-italy/pull/1558/files/d292fd53c935b6bd1f292987d6107aa51cb3206a?file-filters%5B%5D=.js#diff-a922e0c77bfd7ffe58a0e86c06d4ceaaR155-R161

Reply:
> https://github.com/OCA/l10n-italy/pull/1558/files/d292fd53c935b6bd1f292987d6107aa51cb3206a?file-filters%5B%5D=.js#diff-a922e0c77bfd7ffe58a0e86c06d4ceaaR155-R161

Non mi porta sulle righe. Che file? Che righe?

Reply:
Però se togliamo `requested_z_report` poi la stampa del report X, nella `onreceive` viene trattata come se fosse un ordine.
Lo rinominerei allora in `requested_report` e lo userei per entrambe